PendingReleaseNotes

```diff
@@ -133,6 +133,12 @@
   ISA-L is recommended for new pools because the Jerasure library is
   no longer maintained.
 
+* CephFS: Format of name of pool namespace for CephFS volumes has been changed
+  from `fsvolumens__<subvol-name>` to `fsvolumens__<subvol-grp-name>_<subvol-name>`
+  to avoid namespace collision when two subvolumes located in different
+  subvolume groups have the same name. There are no security concerns associated
```

Comment:
sorry, explain a bit more. Something like "Even with namespace collision, there was no security issues since the MDS auth cap is path restricted to the subvolume path. Now, with this change, the namespaces are completely isolated."

Reply:
Done. PTAL.